index.php

```diff
@@ -3,10 +3,42 @@
 require_once __DIR__.'/vendor/autoload.php';
 
 use Heise\Shariff\Backend;
-use Zend\Config\Reader\Json;
 
+/**
+ * Demo Application using Shariff Backend
+ */
 class Application
 {
+    /**
+     * Sample configuration
+     *
+     * Format for this example is JSON for nice readability
+     *
+     * @var string
+     */
+    private static $configuration = '
```

Comment:
Okay, I overlooked

> for nice readability

But is that really the case?
Wouldn't errors caused by misconfiguration be easier to track down if this was a PHP data structure?